.github/scripts/ci_failure_bot.py

```
@@ -0,0 +1,306 @@
+3. **Required Action:** Provide a code block or specific steps the contributor must take.
+4. **Quality Warning (If Applicable):** If the PR appears to be "spam" (e.g., trivial README changes, AI-generated nonsense, or repeated basic errors), include a firm statement that such contributions are a drain on project resources and ping the maintainers asking them for manual review.  # noqa: E501
+
+### EXAMPLE RESPONSE STYLE
+"The build failed because you neglected to update the test suite to match your logic changes. This project does not accept functional changes without corresponding test updates. Refer to the log at line 452. Update tests/logic_test.py before re-submitting. We prioritize high-quality, ready-to-merge code; please ensure you run local tests before pushing."  # noqa: E501
+
+Analyze the failure and provide your response:
+"""
+
+        try:
+            response = self.model.generate_content(context)
+            return response.text
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)